code/modules/mob/living/carbon/human/species.dm

```diff
@@ -362,6 +362,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(DIGITIGRADE in species_traits)
 		C.Digitigrade_Leg_Swap(FALSE)
 
+	if(!(NORIDING in species_traits))
+		AddElement(/datum/element/ridable, /datum/component/riding/creature/human)
```

Comment:
This will stick over species changes. Bad solution and i'd prefer buckling list was made source aware.

Reply:
ah shit good point ok

Reply:
could you elaborate on the second point @AnturK  

_i'd prefer buckling list was made source aware._

Reply:
Something like `whatever.buckled_mobs[buckled] = BUCKLE_TYPE_RIDING`
And then riding/buckling/cuffing interactions only working on these groups.